### DISCUSSION TITLE
[aws-lambda] Property 'identity' does not exist on type 'APIGatewayEventWebsocketRequestContextV2'

### DISCUSSION THREAD

**Discussion by paul-uz** [DefinitelyTyped/DefinitelyTyped#61507] [Issues with a @types package]
Get the TS error

> Property 'identity' does not exist on type 'APIGatewayEventWebsocketRequestContextV2'

When trying to read `event.requestContext.identity.sourceIp. Value is present in the event.requestContext object, verified by logging out the whole event object.

[a comment by an automated account is not shown]

**Comment by jamesorlakin** [comment]
Indeed I think the `identity` section is missing from the current types: https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-mapping-template-reference.html
